services/blockchain-indexer/shared/utils/eventsUtils.js

```diff
@@ -52,6 +74,23 @@ const getEventsInfoToIndex = async (block, events) => {
 	return eventsInfoToIndex;
 };
 
+const deleteEventsTillBlockHeight = async (blockHeight, dbTrx) => {
+	if (Number.isNaN(Number(blockHeight))) return;
```

Comment:
This check isn't required but if left should be like below, since `Number.isNaN(Number(null))` will result in `false`.
```suggestion
	if (typeof blockHeight !== 'number') return;
```

Reply:
Removed.